spec/acceptance/user_journey_spec.rb

```diff
@@ -6,7 +6,11 @@
 
 RSpec.describe 'User journey', type: :feature, acceptance: true do
   let(:root) { PKI.new }
-  let(:cert) { root.generate_encoded_cert(expires_in: 2.months) }
+  let(:certificate) {
+    team = Team.find_or_create_by(name: 'acceptance test team')
+    component = build(:msa_component, entity_id: 'https://acceptance-test-msa-2.com', team_id: team.id )
+    create(:msa_encryption_certificate, component: component )
+  }
```

Comment:
you don't have to do any of this.. this is a test in a browser, it doesn't create any components or anything...